custom_components/better_thermostat/models/models.py

```diff
@@ -4,6 +4,7 @@
 import math
 import os
 from pathlib import Path
+from custom_components.better_thermostat.helpers import get_trv_model
```

Comment:
my IDE cannot resolve the path custom_components.better_thermostat.helpers, but I guess it's fine, right?

Reply:
I fixed it with a relative import path